jenkins-scripts/lib/colcon-default-devel-windows.bat

```diff
@@ -29,8 +29,8 @@ set LOCAL_WS_BUILD=%WORKSPACE%\build
 @if "%COLCON_AUTO_MAJOR_VERSION%" == "" set COLCON_AUTO_MAJOR_VERSION=false
 
 setlocal ENABLEDELAYEDEXPANSION
-if "%COLCON_AUTO_MAJOR_VERSION%" == "true" (
-   for /f %%i in ('python "%SCRIPT_DIR%\tools\detect_cmake_major_version.py" "%WORKSPACE%\%VCS_DIRECTORY%\CMakeLists.txt"') do set PKG_MAJOR_VERSION=%%i
+for /f %%i in ('python "%SCRIPT_DIR%\tools\detect_cmake_major_version.py" "%WORKSPACE%\%VCS_DIRECTORY%\CMakeLists.txt"') do set PKG_MAJOR_VERSION=%%i
+if "%COLCON_AUTO_MAJOR_VERSION%" == "true" || "%PKG_MAJOR_VERSION%" != "1" (
    set COLCON_PACKAGE=%COLCON_PACKAGE%!PKG_MAJOR_VERSION!

```

Comment:
maybe the problem is this line? perhaps we should be trying to detect the cmake project name rather than the major version?

Reply:
I was trying to keep this generic, but in the end of the day this logic seems like it's here only for `ign-tools`, so we could hardcode the name here. Is this what you mean?